### DISCUSSION TITLE
Github -> vercel deploy great what about github -> EAS

### DISCUSSION THREAD

**Discussion by karmatradeDev** [nandorojo/solito#286] [General]
Hi im familiar with deploying to vercel its pretty easy honestly. but id like a guide are tips for deploying solito to expo EAS. anyone have any best practices are guides?

**Comment by MorenoJoshua** [comment]


**Reply by nandorojo** [reply]
hidden

**Reply by MorenoJoshua** [reply]
Removing my reply

**Comment by Silthus** [comment]
Here is my setup for a full automatic CI/CD process using `semantic-release` and `EAS`: https://gist.github.com/Silthus/165cd25e75dd10aacf039caf40664420#file-readme-md

It requires no MAC and automatically builds a new dev client for every push to main.

You must use https://www.conventionalcommits.org/en/v1.0.0/ for the commit messages for the versioning to work.
